docs/ReferenceArrayInput.md

```diff
@@ -328,6 +330,76 @@ Then to display a selector for the post tags, you should call `<ReferenceArrayIn
 <ReferenceArrayInput source="tags_ids" reference="tags" />
 ```
 
+## `readOnly`
+
+If `true`, the default input is in read-only mode and the user can't change the value.
+
+{% raw %}
+
+```jsx
+<ReferenceArrayInput
+    source="tags_ids"
+    reference="tags"
+    readOnly
+/>
+```
+
+{% endraw %}
+
+**Warning**: The `readOnly` prop refer to the default [`<AutocompleteArrayInput>`](./AutocompleteArrayInput.md) child. If you defined `children` you have to specify the `readOnly` prop on each input:
+
+{% raw %}
+
+```jsx
+<ReferenceArrayInput source="tags_ids" reference="tags">
+    <SelectArrayInput readOnly />
+</ReferenceArrayInput>
+```
+
+{% endraw %}
+
+## `disabled`
+
+If `true`, the default input is disabled and the user can't change the value.
```

Comment:
**suggestion**: Same as before, ‏I think something like this could help users to understand the differences between readOnly and disabled props:
```suggestion
The `disabled` prop set to true makes the element not mutable, focusable, or even submitted with the form. The user can neither edit nor focus on the control, nor its form control descendants.
```